src/main/java/dev/pace/staffchat/commands/AdminChatToggle.java

```diff
@@ -18,16 +20,17 @@
     public boolean onCommand(CommandSender commandSender, Command command, String s, String[] strings) {
         if (!(commandSender instanceof Player)) return false;
         Player player = (Player) commandSender;
+        Main main = Main.getInstance();
+        if (main == null) return true;
         if (!player.hasPermission("staff.adminchat") || !player.isOp()) {
-            player.sendMessage(ChatColor.translateAlternateColorCodes('&', Main.config.getString("adminchat.error")));
+            player.sendMessage(ChatColor.translateAlternateColorCodes('&', main.config.getString("adminchat.error")));
             return true;
         }
-
-        Main.getInstance().toggledAC.putIfAbsent(player.getUniqueId(), true);
-
-        boolean isEnabled = Main.getInstance().toggledAC.get(player.getUniqueId());
-        boolean previousValue = Main.getInstance().toggledAC.put(player.getUniqueId(), !isEnabled);
-        player.sendMessage(!previousValue ? ChatColor.translateAlternateColorCodes('&', Main.config.getString("adminchat.toggle-on")) : ChatColor.translateAlternateColorCodes('&', Main.config.getString("adminchat.toggle-off")));
+        UUID uuid = player.getUniqueId();
+        boolean isEnabled = main.toggledAC.get(uuid);
+        main.toggledAC.replace(uuid, isEnabled);
```

Comment:
Won't this just replace with the same values? It should be replacing it with the opposite boolean since you're toggling it.

Reply:
Yes, it should be !

Reply:
Old code was:
```java
boolean isEnabled = Main.getInstance().toggledAC.get(player.getUniqueId());        
boolean previousValue = Main.getInstance().toggledAC.put(player.getUniqueId(), !isEnabled);
```
`.put(...)` returns the previous value before it was set.

For example

If the player in the map is set to `true`, the `.put(...)` will set the boolean to `false` but will return `false` since that is the previous.

We could use `.replace(..)` but make sure to invert the boolean 😄